client/src/Components/TopNav.js

```diff
@@ -18,28 +17,19 @@ class TopNav extends Component {
     window.$('.sidebar-toggle').sideNav({closeOnClick: true})
   }
 
-  weekText (num: number) {
-    if (num === 0) {
-      return 'All'
-    } else {
-      return `Week ${num}`
-    }
-  }
+  renderRightNav () {
+    const { week, weeks, weekChange } = this.props
 
-  renderWeeks (weeks: Array<any>) {
-    return _map(weeks, (week) => {
+    if (week) {
```

Comment:
Selecting "All" from the WeekPicker removes it from the NavBar, because I think All is represented as week 0.

Reply:
you're totally right. I'm not super happy with this refactor yet. I have another idea I'm going to try out.